trio/_core/_io_windows.py

```diff
@@ -341,8 +341,8 @@ def monitor_completion_key(self):
         if sock in self._socket_waiters[which]:
             await _core.checkpoint()
             raise _core.ResourceBusyError(
-                "another task is already waiting to {} this socket"
-                .format(which)
+                "another task is already waiting to {} this socket".
```

Comment:
suggest that unrelated code formatting be done in a separate PR

Reply:
@belm0 Oh, I didn't touch that module at all, guess that came from yapf and it did slip in with a `git add --all` which I learned I shouldn't do. I'll have to check how to revert a single file of a commit to redo this change, I guess that should be possible.